native/consteval.c

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
```

Comment:
Purely FYI: you would usually call this thing a `config header` and this kind of program `gen_config`.
The current name is good too.